resources/v1.18.5/charts/istiod/templates/deployment.yaml

```diff
@@ -75,7 +75,6 @@ spec:
       priorityClassName: "{{ .Values.global.priorityClassName }}"
 {{- end }}
       securityContext:
-        fsGroup: 1337
```

Comment:
securityContext becomes empty. Is this an issue at all?

Reply:
Not an issue. It would be cleaner if we also removed `securityContext`, but it's hard to implement.